src/patient-registration/patient-registration.component.tsx

```diff
@@ -58,6 +62,25 @@ export const PatientRegistration: React.FC = () => {
   const history = useHistory();
   const [identifier, setIdentifier] = useState('');
   const [location, setLocation] = useState('');
+  const [identifierTypes, setIdentifierTypes] = useState(new Array<PatientIdentifierType>());
+  const initialFormValues: FormValues = {
```

Comment:
It looks like something called `initialFormValues` was already being used on line 130/188 below. Where did `initialFormValues` come from before? Why did you introduce this new definition of it?

Reply:
Nice catch @brandones . Seems this was a left over from a rebase against master.

Reply:
Resolved